### ISSUE TITLE
Installation guide question (DB and Wizard)

### ISSUE THREAD

**Issue by vovayatsyuk** [mage-os/devdocs#78]
I have a question about Step 2 and Step 3:

1. [Step 2](https://github.com/mage-os/devdocs/blob/main/installation-guide.md#step-2-configure-the-database) tells to open env.php, but we don't have this file. Do I need to create it?
2. [Step 3](https://github.com/mage-os/devdocs/blob/main/installation-guide.md#step-3-run-the-installation-wizard) tells to run web wizard. Is Web Wizard a thing in Mageos?

Will this work?

---

I've installed Mageos using the bin/magento command from [Magento docs](https://experienceleague.adobe.com/docs/commerce-operations/installation-guide/composer.html?lang=en#install-the-application):

```
bin/magento setup:install \
--base-url=http://mageos.test/ \
--db-host=localhost \
--db-name=mageos \
--db-user=root \
--db-password=password \
--admin-firstname=admin \
--admin-lastname=admin \
--admin-email=admin@admin.com \
--admin-user=admin \
--admin-password=admin123 \
--language=en_US \
--currency=USD \
--timezone=America/Chicago \
--use-rewrites=1 \
--search-engine=opensearch \
--opensearch-host=localhost \
--opensearch-port=9200 \
--opensearch-index-prefix=mageos \
--opensearch-timeout=15
```

**Comment by Genaker** [comment]
the same issue!

**Comment by rhoerr** [comment]
Hi, thanks for the feedback! I believe this was fixed by #82 .
